lib/customer-deposit-wallet/src/Cardano/Wallet/Deposit/Pure/State/Creation.hs

```diff
@@ -60,18 +66,34 @@ data Credentials
     deriving (Generic, Show, Eq)
 
 instance Show XPrv where
-    show = B8.unpack . B16.encode . unXPrv
+    show = B8.unpack . B16.encode . rawSerialiseXPrv
```

Comment:
In `cardano-wallet` we didn't define this instance in the library code for the sake of:
1. not accidentally leaking `show @XPrv`  in production
2. avoiding orphan instances

and instead resorted to implementing it as needed in the spec modules. 

Is there a reason for not doing it like this here?